src/DepthChart.java

```diff
@@ -0,0 +1,200 @@
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Stack;
+
+/**
+ * Javadoc.
+ */
+public class DepthChart implements Iterable<Map.Entry<String, Stack<String>>> {
+    /**
+     * Reptresentation of {@code this}.
+     */
+    private Map<String, Stack<String>> rep;
+
+    /**
+     * Constructor creates an empty depth chart.
+     */
+    public DepthChart() {
+        this.rep = new HashMap<String, Stack<String>>();
+        Stack<String> qb = new Stack<>();
+        this.rep.put("qb", qb);
+        Stack<String> hb = new Stack<>();
+        this.rep.put("hb", hb);
+        Stack<String> fb = new Stack<>();
+        this.rep.put("fb", fb);
+        Stack<String> wr = new Stack<>();
+        this.rep.put("wr", wr);
+        Stack<String> te = new Stack<>();
+        this.rep.put("te", te);
+        Stack<String> ot = new Stack<>();
+        this.rep.put("ot", ot);
+        Stack<String> og = new Stack<>();
+        this.rep.put("og", og);
+        Stack<String> c = new Stack<>();
+        this.rep.put("c", c);
+        Stack<String> dt = new Stack<>();
+        this.rep.put("dt", dt);
+        Stack<String> de = new Stack<>();
+        this.rep.put("de", de);
+        Stack<String> ilb = new Stack<>();
+        this.rep.put("ilb", ilb);
+        Stack<String> olb = new Stack<>();
+        this.rep.put("olb", olb);
+        Stack<String> cb = new Stack<>();
+        this.rep.put("cb", cb);
+        Stack<String> fs = new Stack<>();
+        this.rep.put("fs", fs);
+        Stack<String> ss = new Stack<>();
+        this.rep.put("ss", ss);
+        Stack<String> k = new Stack<>();
+        this.rep.put("k", k);
+        Stack<String> p = new Stack<>();
+        this.rep.put("p", p);
+        Stack<String> ls = new Stack<>();
+        this.rep.put("ls", ls);
+
+    }
+
+    /**
+     * Adds the player to the position (Kernel).
+     */
+    public final void addPlayer(String player, String position) {
+        Stack<String> positionStack = this.rep.get(position);
+        positionStack.push(player);
+    }
+
+    /**
+     * Removes the player from the position (Kernel).
+     */
+    public final String removePlayer(String position) {
+        Stack<String> positionStack = this.rep.get(position);
+        String player = positionStack.pop();
+        return player;
+    }
+
+    /**
+     * Returns the size of one position (Kernel).
+     */
+    public final int numPosition(String position) {
+        Stack<String> positionStack = this.rep.get(position);
+        return positionStack.size();
+    }
+
+    /**
+     * Custom iterator that prints in alphabetical order of positions.
+     */
+    @Override
+    public Iterator<Map.Entry<String, Stack<String>>> iterator() {
+        List<Map.Entry<String, Stack<String>>> sortedEntries = new ArrayList<>(
```

Comment:
Is this `>>` a new bit of syntax? I would assume it is `<>` with a generic type defined.